packages/documents/graphql/schema-types.js

```diff
@@ -64,6 +78,8 @@ type Meta {
   section: Document
   format: Document
   dossier: Document
+  shareImage: String
+  shareImageColor: String
```

Comment:
This won't produce an error but is stilled checked if it is an image URL. Maybe `shareColor`?

Reply:
this one is an image

Reply:
is the type wrong?

Reply:
Oh my! I assumed `shareImageColor` is a hex color code which makes my comment obsolete.